src/router/routes/root.tsx

```diff
@@ -74,22 +72,19 @@ const RootElements = (client: VocdoniSDKClient) => [
     ),
   },
   {
-    element: (
-      <SuspenseLoader>
-        <OrganizationProtectedRoute />
-      </SuspenseLoader>
-    ),
-    children: [
-      {
-        path: Routes.stripe.checkout,
-        element: <StripeCheckout />,
-      },
-      {
-        path: Routes.stripe.return,
-        element: <StripeReturn />,
-        errorElement: <Error />,
-      },
-    ],
+    ...ProtectedRoutes({
+      children: [
+        {
+          path: Routes.stripe.checkout,
+          element: <StripeCheckout />,
+        },
+        {
+          path: Routes.stripe.return,
+          element: <StripeReturn />,
+          errorElement: <Error />,
+        },
+      ],
+    }),
```

Comment:
Remember to also change this one if you end up changing the ProtectedRoutes method definition.

Reply:
b18a0c9